packages/yourloops/pages/profile/auth-form.tsx

```diff
@@ -86,32 +81,18 @@ function AuthenticationForm(props: AuthenticationFormProps): JSX.Element {
         helperText={t("no-password")}
         onChange={setCurrentPassword}
       />
-      <Password
-        id="profile-textfield-password"
-        autoComplete="new-password"
-        variant="standard"
-        label={t("new-password")}
-        value={password}
-        error={errors.password && password.length > 0}
-        checkStrength
-        helperText={
-          <PasswordStrengthMeter
-            force={passwordCheckResults.score}
-            error={errors.password}
-            helperText={passwordCheckResults.helperText}
-          />
-        }
-        onChange={setPassword}
-      />
-      <Password
-        id="profile-textfield-password-confirmation"
-        autoComplete="new-password"
-        variant="standard"
-        label={t("confirm-password")}
-        value={passwordConfirmation}
-        error={errors.passwordConfirmation}
-        helperText={t("not-matching-password")}
-        onChange={setPasswordConfirmation}
+      <PasswordConfirm
+        variant={"standard"}
+        onError={(password, passwordConfirmation) => {
+          setPassword(password);
+          setPasswordConfirmation(passwordConfirmation);
+          setPasswordConfirmationError(true);
+        }}
+        onSuccess={(password) => {
+          setPasswordConfirmation(password);
+          setPassword(password);
+          setPasswordConfirmationError(false);
+        }}
```

Comment:
Can we prefer method like "onBlur" in password-confirm component
```suggestion
      <PasswordConfirm
        variant={"standard"}
        onError={onError}
        onSuccess={onSuccess}
```